src/app/sharedUI/items/ArtworkGridItem.tsx

```diff
@@ -33,12 +33,12 @@ export const ArtworkGridItem: React.FC<ArtworkGridItemProps> = (props) => {
           </Text>
         </Text>
       </Flex>
-      {!disable && selectedToAdd ? (
+      {!disable && selectedToAdd && (
         <Flex position="absolute" top={1} right={1} alignItems="center" justifyContent="center">
           <CheckCircleFillIcon height={30} width={30} fill="blue100" />
         </Flex>
-      ) : null}
-      {selectedToRemove ? (
+      )}
+      {selectedToRemove && (
```

Comment:
this is not a strict boolean, so we should not do that here.

Reply:
actually, maybe we can quickly enable the eslint rule for this? let me find the one i mean.

Reply:
this would be ok if we do 
```suggestion
      {selectedToRemove === true && (
```